src/handlers/deeplinks.ts

```diff
@@ -195,10 +199,13 @@ function handleWalletConnect(uri: any) {
     parsedUri.version === 2 &&
     getExperimetalFlag(WC_V2)
   ) {
+    logger.debug(`handleWalletConnect: handling v2`, { uri });
     setHasPendingDeeplinkPendingRedirect(true);
     pairWalletConnect({ uri });
   } else {
+    logger.debug(`handleWalletConnect: handling fallback`, { uri });
     // This is when we get focused by WC due to a signing request
+    setHasPendingDeeplinkPendingRedirect(true);
```

Comment:
Not actually hitting this fork atm, but IF a dapp integrates more deeply I believe they could use universal links to trigger this, so this will handle mobile redirection as it should IF that ever happens.